include/zephyr/dt-bindings/input/input-event-codes.h

```diff
@@ -26,6 +26,7 @@
 #define INPUT_EV_REL 0x02               /**< Relative coordinate event */
 #define INPUT_EV_ABS 0x03               /**< Absolute coordinate event */
 #define INPUT_EV_MSC 0x04               /**< Miscellaneous event */
+#define INPUT_EV_DEVICE 0xef            /**< Specific input device event */
```

Comment:
here again, "Device specific input event"?
Also questioning whether `INPUT_EV_DEVICE` is really appropriate, it sounds like it's missing a qualifier somehow, but I also understand the intent of trying to keep the event type short, so not really a blocker

Reply:
It seems we have an agreement to let this one as is for keeping the name of the codes short as possible.

Reply:
yes but please fix the doxygen comment to say "Device specific input event". Thanks!

Reply:
Done, thanks for  reviewing that folks.